lms/djangoapps/certificates/tests/test_cert_management.py

```diff
@@ -35,11 +36,12 @@ def _create_cert(self, course_key, user, status):
         # Enroll the user in the course
         CourseEnrollmentFactory.create(
             user=user,
-            course_id=course_key
+            course_id=course_key,
+            mode=CourseMode.HONOR
```

Comment:
With the "honor" mode effectively going away, I wonder if we should also test other modes, such as Audit and Verified?